spec/unit/parser/functions_spec.rb

```diff
@@ -48,8 +48,14 @@ def callable_functions_from(mod)
   end
 
   describe "when calling function to test function existence" do
+    it "should loadall 3x functions" do
+      Puppet::Parser::Functions.autoloader.delegatee.stubs(:loadall)
+
+      Puppet::Parser::Functions.autoloader.loadall
+    end
```

Comment:
Shouldn't there be an assertion in this test? - This looks like it is only doing a stub which gets triggered via loadall, but then there is nothing else...